### DISCUSSION TITLE
Schema types end with a '2'

### DISCUSSION THREAD

**Discussion by michaelhthomas** [MichalLytek/type-graphql#1406] [Q&A]
This is probably something silly, but I can's seem to figure it out. Using version `2.0.0-beta.1`, all of the object, input, and interface types in my schema have a 2 at the end after being converted to SDL. Is this intended behavior, and if not, what could be causing it?

**Comment by MichalLytek** [comment]
I have not experienced this and I wouldn't blame TypeGraphQL.

> after being converted to SDL

What tool are you using for that?

**Reply by michaelhthomas** [reply]
Just enabled the type-graphql emit option and looked at the resulting file. I'm using esbuild with the decorators plugin if that could have anything to do with it.

**Reply by MichalLytek** [reply]
It could, show the transpiled code.

**Reply by michaelhthomas** [reply]
Looks like that was it, esbuild was appending numbers to avoid namespace conflicts. Looks like there's a way around this with the `--keep-names` option, see [here](https://github.com/evanw/esbuild/issues/510). Thanks for your help!
